src/pages/FarmV3/ActionModal.tsx

```diff
@@ -92,170 +72,153 @@ export const ActionModal: FC<{
 
   const Content = useCallback(
     () => (
-      <div>
-        {breakpoint.isMobile && (
+      <div className="w-full h-full flex flex-col justify-between">
+        <div>
           <div
-            tw="absolute right-[25px] text-[25px] cursor-pointer"
+            className="absolute right-[25px] text-[25px] cursor-pointer"
             onClick={() => {
```

Comment:
Not your code, but this is better to memoize too

Reply:
its already memoized in the callback, but the callback should probably be a memo